sklearn/metrics/_plot/tests/test_common_curve_display.py

```diff
@@ -0,0 +1,138 @@
+import pytest
```

Comment:
Note to reviewer: this test will be used as well for the `RocCurveDisplay` and the `DetCurveDisplay`